tangelo/toolboxes/qubit_mappings/combinatorial.py

```diff
@@ -40,8 +39,155 @@
 from tangelo.toolboxes.operators import QubitOperator
 
 
+ZERO_TOLERANCE = 1e-8
+
+
+def int_to_tuple(integer, n_qubits):
+    """ Convert integer encoding a qubit Hamiltonian term in stabilizer representation
```

Comment:
Convert a qubit Hamiltonian term in integer encoding (stabilizer representation) into an Openfermion-style tuple.